docs/Backtracing.rst

```diff
@@ -229,3 +259,57 @@ Other Darwin (iOS, tvOS)
 
 Crash catching is not enabled for non-macOS Darwin.  You should continue to look
 at the system-provided crash logs.
+
+Linux
+^^^^^
+
+Frame Pointers
+""""""""""""""
+
+The backtracer currently does a simple frame-pointer based unwind.  As a result,
+if you compile your code with ``-fomit-frame-pointer``, which is often the
+default for release builds on Intel Linux, you may find that you get incomplete
+backtraces.
+
+If you wish to get a more complete backtrace, at a small cost in performance,
+you can add the compiler flags ``-Xcc -fno-omit-frame-pointer`` when building
+your Swift program.
+
+Static Linking Support
+""""""""""""""""""""""
+
+For users who statically link their binaries and do not wish to ship the Swift
+runtime library alongside them, there is a statically linked copy of
+``swift-backtrace``, named ``swift-backtrace-static`` , in the ``libexec``
+directory alongside the normal ``swift-backtrace`` binary.
+
+By default, to locate ``swift-backtrace``, the runtime will attempt to look in
+the following locations::
+
+    <swift-root>/libexec/swift/<platform>
+    <swift-root>/libexec/swift/<platform>/<arch>
```

Comment:
Will people know the exact form that `<platform>` and `<arch>` will have? If not, would be good to briefly describe that.

Reply:
I've added a description.